### ISSUE TITLE
AVT: solicita envio de notificaciones a correo

### ISSUE THREAD

**Issue by rpardines** [phicus/krill#108]
Desde AVT nos solicitan que las alarmas de los siguientes ejemplos, lleguen a un email:

OLT01-SOR00 PROBLEM  PING CRITICAL - Packet loss = 100% loc:SOR [avm](http://avm.phicus.es:4267/host/OLT01-SOR00)

GPON8-Hola/onug-olt0/01/04 PROBLEM  New alarmed ONUs (11) greater than 5! Time to expire alarm: 40.00 hours
